### PR TITLE
1.20.41--FeaturedOffers

### PR DESCRIPTION
**Pull request by rickeydute**
-FO page is on template__b
-Added code for a collapsible sticky menu for the FO small tablet and mobile page.
-Changed the size of the Nosto carets from 35px x 84px to 12px x 23px.
-Created a new class to override some Nosto stylings.
-Removed the lazy-load from the Clyde modal to force the images to load on cart load.
-Removed a sentence from the 404 page per Alix's request.

### CODE REVIEW

File: assets/scss/components/_generics.scss [hunk 1]
```diff
@@ -614,6 +620,14 @@
     }
 }
 
+.navHeight-2{
```
Comment (teamteak) [inline assets/scss/components/_generics.scss:623]: This is not a meaningful class name.  why are there two?

File: assets/scss/components/_generics.scss [hunk 2]
```diff
@@ -606,6 +606,12 @@
     top: 130px;
 }
 
+.sticky-2{
```
Comment (teamteak) [inline assets/scss/components/_generics.scss:609]: This is not a meaningful class name.  why are there two?

**Review by teamteak — APPROVED** [review]

File: assets/scss/components/_generics.scss (hunk 2 above)
Comment (rickeydute) [reply assets/scss/components/_generics.scss:609]: navHeight was added to work with the sticky menu on the Shipping Policy page.
navHeight-2 was added for the sticky menu for the Featured Offers page. 
Unfortunately, the Shipping Policy is so tall that using this on the FO page makes the page very long and adds a lot of extra whitespace after the content is finished so a new class to accommodate the length of the FO page.

Comment (teamteak) [reply assets/scss/components/_generics.scss:609]: the classes need to convey meaning, aka, what does the class mean? adding a -2 isn't a good way to create classes as it doesn't indicate what its use would be more where it's used.

File: assets/scss/templates/_product.scss [hunk 3]
```diff
@@ -1005,6 +1005,18 @@
 
 }
 
+#pdf a:link {
```
Comment (teamteak) [inline assets/scss/templates/_product.scss:1008]: We don't style IDs .  Only classes

Comment (teamteak) [reply assets/scss/templates/_product.scss:1008]: This needs to be fixed

**Review by teamteak — CHANGES_REQUESTED** [review]
Remove the styles for the ID and change it to meaningful classes.

File: assets/scss/components/_generics.scss (hunk 2 above)
Comment (rickeydute) [reply assets/scss/components/_generics.scss:609]: Ok. I'll keep that in mind for the next time. I'll switch them and release an update at the end of the sprint to not hold up this release any longer.

**Review by teamteak — APPROVED** [review]

**Pull request merged**